tests/output_config_tests/expected_output_files/basic_semantic_diff_yaml__stdout_output.txt

```diff
@@ -12,15 +12,267 @@ global: cluster has 29 unique endpoints, 8 namespaces
     dst_pods:
     - '*'
     connection:
+    - Protocol: ICMP
+    - Protocol: 2
+    - Protocol: 3
+    - Protocol: 4
+    - Protocol: 5
     - Protocol: TCP
       Ports:
       - 1-52
       - 54-65535
+    - Protocol: 7
+    - Protocol: 8
+    - Protocol: 9
+    - Protocol: 10
+    - Protocol: 11
+    - Protocol: 12
+    - Protocol: 13
+    - Protocol: 14
+    - Protocol: 15
+    - Protocol: 16
```

Comment:
We definitely need protocol ranges.
Also, checking if complement is shorter?

Reply:
done